kxc3244/playground
# kevin chawla

Description 
what did you created or change or fix

Steps to test

1. npn install
2. npm run start
3. Go to the search 
4. Verify on typing the auto suggest
